interface/orders/procedure_provider_edit.php

```diff
@@ -38,279 +38,426 @@ function invalue($name)
 ?>
 <html>
 <head>
+<?php Header::setupHeader(['jquery-ui', 'jquery-ui-base']);?>
 <script type="text/javascript" src="<?php echo $webroot ?>/interface/main/tabs/js/include_opener.js"></script>
```

Comment:
drop above line and add `opener` to the above setupHeader call

Reply:
need jquery-ui for jquery tooltips

Reply:
to clarify, meant to drop the include_opener.js include and then do:
```
<?php Header::setupHeader(['jquery-ui', 'jquery-ui-base','opener']);?>
```

Reply:
Done